webapp/index.html

```diff
@@ -1,11 +1,14 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="">
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico" />
-    <title><%= htmlWebpackPlugin.options.title %></title>
+    <link rel="icon" href="/favicon.ico" />
+    <title>Default Title</title>
```

Comment:
```suggestion
    <title>datalab</title>
```

May as well just use datalab here too, or can this be removed entirely?